saleor/graphql/shipping/types.py

```diff
@@ -99,30 +100,42 @@ class ShippingMethod(ChannelContextTypeWithMetadata, CountableDjangoObjectType):
         "saleor.graphql.product.types.products.Product",
         description="List of excluded products for the shipping method.",
     )
+    id = graphene.ID(required=True, description="Shipping method ID.")
+    name = graphene.String(required=True, description="Shipping method name.")
+    description = graphene.JSONString(description="Shipping method description (JSON).")
```

Comment:
I don't think that we need to add `(JSON)` here. We use `JSONString` type which informs you that this is a JSON field